pkg/config/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	SumDBs           []string `envconfig:"ATHENS_SUM_DBS"`
 	NoSumPatterns    []string `envconfig:"ATHENS_GONOSUM_PATTERNS"`
 	SingleFlightType string   `envconfig:"ATHENS_SINGLE_FLIGHT_TYPE"`
+	RobotsFile       string   `envconfig:"ROBOTS_FILE"`
```

Comment:
this needs to be propagated in a few places: `config.dev.toml`, `helm` and `defaultConfigs`